nbs/core.ipynb

```diff
@@ -54,10 +54,15 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.                print('Using stored dataset.')
</pre><p>Could we change the print to&nbsp;<code>warnings.warn</code>? To prevent incompatibilities with loggers.</p><p> </p><p>Here's an example:</p><pre class="ql-syntax" spellcheck="false">import warnings

...

warnings.warn('Using stored dataset.')
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
no, will use logger